.travis.yml

```diff
@@ -3,14 +3,20 @@ python: 2.7
 sudo: false
 env:
   matrix:
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=pypy
-  - TOX_ENV=docs
-  - TOX_ENV=system-tests
-  - TOX_ENV=system-tests3
+  # - TOX_ENV=py26
+  # - TOX_ENV=py27
+  # - TOX_ENV=py33
+  # - TOX_ENV=py34
+  # - TOX_ENV=pypy
+  # - TOX_ENV=docs
+  # - TOX_ENV=system-tests
+  # - TOX_ENV=system-tests3
```

Comment:
You're gonna want to uncomment then before the merge (I'm sure you're aware, just don't want to forget).